server/db.js

```diff
@@ -1,19 +1,62 @@
-// const mongoose = require('mongoose');
-// mongoose.Promise = require('bluebird');
-// require('dotenv').config();
-
-// const databaseName = process.env.DB_NAME;
-// const databaseHost = process.env.DB_HOST || 'localhost';
-
-// const handleConnect = () => {
-//   if (mongoose.connection.readyState === 0) { // not connected to database
-//     return mongoose.connect(`mongodb://${databaseHost}:27017/${databaseName}`, { useNewUrlParser: true, useCreateIndex: true });
-//   }
-//   return Promise.resolve(false); // already connected
-// };
-
-// module.exports = {
-//   databaseName,
-//   mongoose,
-//   handleConnect,
-// };
+const Sequelize = require('sequelize');
+
+const sequelize = new Sequelize('postgres://postgres@localhost:5432/bidbuy');
+const Model = Sequelize.Model;
+
+class Item extends Model {};
+
+Item.init({
+  id: {
+    type: Sequelize.INTEGER,
+    primaryKey: true,
+    autoIncrement: true
+  },
+  name: Sequelize.STRING,
+  url: Sequelize.STRING,
+  condition: Sequelize.STRING,
+  price: Sequelize.DECIMAL,
+  sellerNote: Sequelize.STRING,
+  expiresAt: Sequelize.DATE,
+  watchers: Sequelize.INTEGER,
+  bids: Sequelize.INTEGER,
+  shippingCountry: Sequelize.STRING,
+  returnsAllowed: Sequelize.BOOLEAN,
+}, { sequelize, modelName: 'item' });
+
+
+const getById = (id) => {
+  return Item.findOne({ raw: true, where: { id: id }, benchmark: true });
```

Comment:
I noticed you used pgcli in your SS for timing- did you mean to keep the benchmark: true in all of these queries as well?